src/serlo-editor/plugins/text/utils/insert-plugin.ts

```diff
@@ -0,0 +1,75 @@
+import { Action, ThunkDispatch } from '@reduxjs/toolkit'
+import { Editor as SlateEditor, Node } from 'slate'
+
+import { sliceNodesAfterSelection } from './document'
+import {
+  RootStore,
+  insertPluginChildAfter,
+  runReplaceDocumentSaga,
+  selectDocument,
+  selectMayManipulateSiblings,
+  selectParent,
+} from '@/serlo-editor/store'
+
+export interface insertPluginArgs {
+  pluginType: string
+  editor: SlateEditor
+  store: RootStore
+  id: string
+  dispatch: ThunkDispatch<unknown, unknown, Action<unknown>>
+  state?: unknown
+}
+
+export function insertPlugin({
```

Comment:
maybe "split" should be part of the name since it splits the text plugin and then adds a new editor plugin, but I'm not sure if that is needed.